app/src/main/java/com/keylesspalace/tusky/components/scheduled/ScheduledStatusAdapter.kt

```diff
@@ -54,7 +54,6 @@ class ScheduledStatusAdapter(
             holder.binding.delete.isEnabled = true
             holder.binding.text.text = item.params.text
             holder.binding.edit.setOnClickListener { v: View ->
```

Comment:
```suggestion
            holder.binding.edit.setOnClickListener {
```

Remove unnecessary parameter.